Source/santactl/Commands/sync/SNTCommandSyncManager.m

```diff
@@ -47,7 +47,11 @@ @interface SNTCommandSyncManager () {
 @property(nonatomic) dispatch_source_t ruleSyncTimer;
 
 @property(nonatomic) NSCache *dispatchLock;
-@property(nonatomic) NSCache *ruleSyncCache;
+
+// Changed this to an NSMutableDictionary instead of NSCache so that we can iterate over its keys
```

Comment:
Sorry, I was using a lot of these comments as temporary documentation.  I'll go ahead and fix them.  Let me know if they still seem weird / too verbose.